Assets/_Functionalities/ObjectInformation/Scripts/ObjectSelectorService.cs

```diff
@@ -150,8 +178,13 @@ private void Update()
                         lastSelectedMappingLayerData = null;
                     }
                     if (mapping is MeshMapping map)
-                    {                      
+                    {
                         LayerData layerData = subObjectSelector.GetLayerDataForSubObject(map.ObjectMapping);
+                        //TODO maybe to the best place here to have a dependency to the cartesianlayerstyler, needs a better implementation
+                        LayerFeature feature = GetLayerFeatureFromBagID(bagId, map, out LayerGameObject layer);
```

Comment:
Feature mag null zijn, maar hij wordt wel hieronder doorgegeven, dat is geen probleem? Of moet hier een nullcheck komen?